packages/std/src/math/uint256.rs

```diff
@@ -48,13 +39,13 @@ use uints::U256;
 /// assert_eq!(a, b);
 /// ```
 #[derive(Copy, Clone, Default, Debug, PartialEq, Eq, PartialOrd, Ord, JsonSchema)]
-pub struct Uint256(#[schemars(with = "String")] U256);
+pub struct Uint256(#[schemars(with = "String")] pub(crate) U256);
```

Comment:
Do we really need this `pub(crate)`?

Reply:
Good point, we don't need it right now, but it will be needed when implementing `Int256` (for `abs_diff`)